app/components/e-panel.vue

```diff
@@ -0,0 +1,543 @@
+<template>
+  <div :class="b({ contour: contour, color: color })">
+    <!-- TODO replace with e-headline -->
+    <!-- eslint-disable max-len -->
+    <div v-if="hasHeadline" :class="b('headline', { 'spacing-top': headlineSpacingTop, 'spacing-right': headlineSpacingRight, 'spacing-bottom': headlineSpacingBottom, 'spacing-left': headlineSpacingLeft })">
+      <span :class="b('headline-underline')">{{ headline }}</span>
+    </div>
+    <!-- eslint-disable max-len -->
+    <div :class="b('content', { 'spacing-top': contentSpacingTop, 'spacing-right': contentSpacingRight, 'spacing-bottom': contentSpacingBottom, 'spacing-left': contentSpacingLeft })">
+      foo<br>bar<br>
+      foo<br>bar<br>
+    </div>
+    <div
+      v-if="triangleButton"
+      :class="b('triangle-button', { size: triangleButtonSize, contour: contour, color: color })"
+      @click="$emit('triangle-pressed')"
+      @mouseover="mouseOver"
+      @mouseout="mouseOut">
+      <span :class="b('plus', { size: triangleButtonSize, 'plus-color': triangleButtonPlusColor })"></span>
+    </div>
+  </div>
+</template>
+
+<script>
+  export default {
+    name: 'e-panel',
+    // components: {},
+    // mixins: [],
+
+    props: {
```

Comment:
- [ ] There should also be an `active` or `open` state prop which forces the `x` because it can be toggle for dropdowns and so on.